rails/config/routes.rb

```diff
@@ -548,6 +570,8 @@
     get '/time' => 'misc_metal#time', :as => :time
     get '/learner_proc_stats' => 'misc#learner_proc_stats', :as => :learner_proc_stats
     get '/learner_proc' => 'misc#learner_proc', :as => :learner_proc
+    get '/preflight' => 'misc#preflight', :as => :preflight
+    get '/stats' => 'misc#stats', :as => :stats
```

Comment:
I don't think we should change the path to these routes since at least the preflight route is configured in Schoology.
So they should be `/misc/preflight` and `/misc/stats`

Reply:
I didn't see a reference to it in the code and I just checked cloudformation and don't see preflight referenced.  Where is that setup at?

Reply:
I've added `/misc` to these routes.

Reply:
The setup is in Schoology itself. It